aemedge/templates/blog-article/blog-article.css

```diff
@@ -247,6 +247,24 @@ a.button.primary:hover
     cursor: none;
 }
 
+.blog-article .default-content-wrapper p,
+.blog-article .default-content-wrapper ul,
+.blog-article .default-content-wrapper li {
+    text-wrap: initial;
```

Comment:
If you think this is a universal ask, you can remove "text-wrap: balance" from styles.css line 137.

Reply:
added this change to code